### ISSUE TITLE
Add support for `FILL_BYTES(n, value)`

### ISSUE THREAD

**Issue by felipesanches** [felipesanches/AnotherWorld_VMTools#21]
That would make the final portion of each disassembled file cleaner, as we currently fill the rest of the 64k address space with 0xFF bytes, which results in a very long and annoying tail of `db 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF`

![Screenshot from 2023-01-21 03-42-41](https://user-images.githubusercontent.com/213676/213841970-e15ebbe5-f594-43e4-995d-c15129b1b015.png)

This could instead be something like: `FILL_BYTES(8237, 0xFF)` (in a hypothetical case of 8237 0xFF bytes in the end of a bytecode binary.

**Comment by Joao620** [comment]
This function would work anywhere in the code that have N repeating bytes or just in the final padding of the file?

**Comment by felipesanches** [comment]
yeah... anywhere in the code.

**Comment by Joao620** [comment]
So that would be a change in the ExecTrace library right? I didn't see any mention of 'db' in the dissembler, and i don't see a way to have the this control on the subclass

**Comment by felipesanches** [comment]
Yes, exactly! I should have opened this issue in that other repo, indeed.
